### DISCUSSION TITLE
关于 docker build 过程中将服务器卡死 || About the server being stuck during the docker build process

### DISCUSSION THREAD

**Discussion by siyongbo** [walinejs/waline#2608] [Q&A] [answered]
能否讲镜像直接上传到 docker hub 中呢供使用者直接 pull 呢，自己构建直接把服务器卡死了，CPU 、内存占用率都 100% 

---
Can the image be uploaded directly to docker hub for users to pull directly? Building it by yourself will directly freeze the server, and the CPU and memory usage will be 100%

**Comment by lizheming** [comment, marked as answer]
看下文档
https://waline.js.org/guide/deploy/vps.html
https://hub.docker.com/r/lizheming/waline

---
Take a look at the documentation
https://waline.js.org/guide/deploy/vps.html
https://hub.docker.com/r/lizheming/waline

**Reply by siyongbo** [reply]
谢谢，因 docker 官方调整，国内镜像挂掉，感谢你的答复

---
Thank you. Due to the official adjustment of docker, the domestic image is down. Thank you for your reply
